### ISSUE TITLE
Should include default LWRP for rvm installation

### ISSUE THREAD

**Issue by thedeeno** [sous-chefs/chef-rvm#108]
Consumers should be able to install rvm with a LWRP. Currently, we must install rvm with one of the included recipes. Only after this do we have the flexibility to manage rvm with the included LWRPs. If one prefers LWRPs, it makes sense to support installing rvm with a LWRP too.

**Comment by cmluciano** [comment]
This is in progress for the next major release.

**Comment by damacus** [comment]
Closing due to inactivity. 

If this is still an issue please reopen or open another issue. Alternatively drop by the #sous-chefs channel on the [Chef Community Slack](http://community-slack.chef.io/) and we'll be happy to help! 

Thanks,
Sous-Chefs
